packages/components/src/List/ListItem.module.css

```diff
@@ -1,18 +1,28 @@
 .action {
-  display: flex;
   width: 100%;
-  box-sizing: border-box;
-  padding: var(--space-small);
+  padding: 0;
   border: none;
-  text-align: left;
-  text-decoration: none;
   background-color: transparent;
-  transition: background-color var(--timing-base);
```

Comment:
just noticing this is gone now

gonna check vs master to see if the background color timing is noticeable different now, or if this was strictly needed for `.action` when it was on the `button`

Reply:
Why is this gone now?

Reply:
confirmed that we do in fact need it. must have gotten lost in the shuffle.

we want that all the time no matter what, since it's part of the hover/focus styles so I think we need to apply it on a class that is always there.

```
.action > * {
  transition: background-color var(--timing-base);
}

```